tests/unittests/PalTests.cpp

```diff
@@ -1,370 +1,16 @@
-#if 0
 //
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // SPDX-License-Identifier: Apache-2.0
 //
 
 #include "common/Common.hpp"
-#include "common/MockISemanticContext.hpp"
+#include "pal/PseudoRandomGenerator.hpp"
+#include "Version.hpp"
```

Comment:
You shouldn't be using `Version.hpp` anywhere.

Reply:
Is there a test validating SDK version? If so, this header is fine to have here. If the test is removed, you don't need this.

Reply:
Yep. The last one in PalTests.cpp. Thats why I added it.